modin/backends/base/query_compiler.py

```diff
@@ -1740,218 +3577,1371 @@ def set_index_names(self, names, axis=0):
+        )
+
+    @_doc_window_method(
+        win_type="window of the specified type",
+        result="standart deviation",
+        method="std",
+        params="""ddof : int, default: 1
+        *args : iterable
+        **kwargs : dict""",
+    )
+    def window_std(self, window_args, ddof=1, *args, **kwargs):
+        return RollingDefault.register(pandas.core.window.Window.std)(
+            self, window_args, ddof, *args, **kwargs
+        )
+
+    @_doc_window_method(
+        win_type="window of the specified type",
+        result="sum",
+        method="sum",
+        params="""*args : iterable
+        **kwargs : dict""",
+    )
+    def window_sum(self, window_args, *args, **kwargs):
+        return RollingDefault.register(pandas.core.window.Window.sum)(
+            self, window_args, *args, **kwargs
+        )
+
+    @_doc_window_method(
+        win_type="window of the specified type",
+        result="variance",
+        method="var",
+        params="""ddof : int, default: 1
+        *args : iterable
+        **kwargs : dict""",
+    )
+    def window_var(self, window_args, ddof=1, *args, **kwargs):
+        return RollingDefault.register(pandas.core.window.Window.var)(
+            self, window_args, ddof, *args, **kwargs
+        )
 
     # End of Window methods
 
     # Categories methods
 
-    cat_codes = CatDefault.register(pandas.Series.cat.codes)
+    @_add_one_column_warning
+    @add_refer_to("Series.cat.codes")
+    def cat_codes(self):
+        """
+        Convert underlying categories data into its codes.
+
+        Returns
+        -------
+        BaseQueryCompiler
+            New QueryCompiler containing the integer codes of the underlying
+            categories.
+        """
+        return CatDefault.register(pandas.Series.cat.codes)(self)
 
     # End of Categories methods
 
     # DataFrame methods
 
-    invert = DataFrameDefault.register(pandas.DataFrame.__invert__)
-    mad = DataFrameDefault.register(pandas.DataFrame.mad)
-    kurt = DataFrameDefault.register(pandas.DataFrame.kurt)
+    def invert(self):
+        """
+        Apply bitwise invertion for each element of the QueryCompiler.
+
+        Returns
+        -------
+        BaseQueryCompiler
+            New QueryCompiler containing bitwise invertion for each value.
+        """
+        return DataFrameDefault.register(pandas.DataFrame.__invert__)(self)
+
+    @_doc_reduce_agg(
+        method="mean absolute deviation",
+        params="""axis : {0, 1}
+        skipna : bool
+        level : None, default: None
+            Serves the compatibility purpose. Always have to be None.""",
+        link="mad",
+    )
+    def mad(self, axis, skipna, level=None):
+        return DataFrameDefault.register(pandas.DataFrame.mad)(
+            self, axis=axis, skipna=skipna, level=level
+        )
+
+    @_doc_reduce_agg(
+        method="unibased kurtosis", link="kurt", extra_params=["skipna", "**kwargs"]
```

Comment:
why it does not describe all the parameters?

Reply:
it describes all parameters, part of the docs are hidden in the list of default parameters inside [`doc_reduce_agg`](https://github.com/dchigarev/modin/blob/e3f0d9a0c9554acaa1bfabf5c46e11984d36a3e5/modin/backends/base/doc_utils.py#L244)